pii_recognition/pipelines/pii_validation_pipeline.py

```diff
@@ -76,18 +77,54 @@ def calculate_aggregate_metrics(
     return results
 
 
-@returns
+@returns()
 def report_results(results: Dict[str, float], dump_file: str):
     dump_to_json_file(results, dump_file)
 
 
+# TODO: F1 is an accurate name, change to fbeta
 def get_rollup_f1s_on_pii(
     scores: List[TextScore], f1_beta: float, recall_threshold: Optional[float]
 ) -> List[float]:
     f1s = []
     for text_score in scores:
         precisions = [p.precision for p in text_score.precisions]
         recalls = [r.recall for r in text_score.recalls]
-        f1 = compute_pii_detection_f1(precisions, recalls, recall_threshold, f1_beta)
+
```

Comment:
Since this is a bug-fix, I'll create another PR to deal with this instead of putting it here as part of integration test.